src/containers/PrimeStatusBanner/index.tsx

```diff
@@ -137,11 +143,17 @@ export const PrimeStatusBannerUi: React.FC<PrimeStatusBannerUiProps> = ({
   return (
     <Card
       className={cn(
-        'flex flex-col content-center md:flex-row md:justify-between',
+        'relative flex flex-col content-center md:flex-row md:justify-between',
         isUserXvsStakeHighEnoughForPrime && 'items-start md:items-center',
         className,
       )}
     >
+      {shouldShowPrimeTokensLeftIndicator && (
+        <div className="absolute top-0 -mt-4 max-sm:right-4 sm:left-18 md:left-20">
```

Comment:
I'd recommend sticking to a mobile-first only approach to avoid confusion:
```suggestion
        <div className="absolute top-0 -mt-4 right-4 sm:right-0 sm:left-18 md:left-20">
```

Reply:
If we do `sm:right-0`, this would mean adding `right: 0` instead of just not having it above a certain breakpoint

<img width="1029" alt="Screen Shot 2023-10-17 at 0 34 46" src="https://github.com/VenusProtocol/venus-protocol-interface/assets/52756735/0d0e474f-2291-4666-8b26-318b74eecf3a">

Reply:
You're right, my bad:
```recommendation
<div className="absolute top-0 -mt-4 right-4 sm:right-auto sm:left-18 md:left-20">
```